src/components/RecipeView.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Pagination from "./Pagination";
+import Recipe from "./Recipe";
+import "../styles/RecipeView.css";
+
+class RecipeView extends React.Component {
```

Comment:
Add a comment at the beginning detailing what view exactly this is - we now have Recipe, RecipeView, etc, which is confusing what view is what.